### ISSUE TITLE
formatting specifier missing in rust-mode

### ISSUE THREAD

**Issue by jcs090218** [Artawower/turbo-log#16]
It seems like the output is missing a formatting specifier in `rust-mode`:

```rust
let ch: char = dna::new_char();
println!("TCL: [line 39][main.rs] ch: ", ch);
```

should be:

```rust
let ch: char = dna::new_char();
println!("TCL: [line 39][main.rs] {} ", ch);
```

**Comment by Artawower** [comment]
Thank you for the bug report. Fixed in 2.2.2

**Comment by jcs090218** [comment]
That was quick! Thank you so much! :)
